documentation/book/watching-multiple-namespaces.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+// Base the file name and the ID on the module title. For example:
+// * file name: assembly-cluster-operator.adoc
+// * ID: [id='cluster-operator-{context}']
+// * Title: = Configure Cluster Operator
+
+// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+
+include::common/attributes.adoc[]
+
+[id='watching-multiple-namespaces-{context}']
+= Configuring Cluster Operator to watch multiple namespaces
+
+The `STRIMZI_NAMESPACE` environment variable can be used to configure a single operator instance
+to operate in multiple namespaces. For each namespace given, the operator will watch for cluster ConfigMaps
+and perform periodic reconciliation. To be able to do this, the operator's `ServiceAccount` needs
+access to the necessary resources in those other namespaces. This can be done by creating an additional
+RoleBinding in each of those namespaces, associating the operator's ServiceAccount
+(`strimzi-cluster-operator` in the examples) with the operator's
+Role (`strimzi-operator-role` in the examples).
+
+.Procedure
+
+//Add a procedure to explain how to configure operator to watch multiple name spaces
+
+In the following example, the operator deployed in namespace `foo` and needs to operate in namespace `bar`.
+The following RoleBinding would grant the necessary permissions:
+
+.Example RoleBinding for a operator to operate in namespace `bar`
```

Comment:
Replace with _.Example `Deployment` for an operator watching in bamespace `bar` and `baz`_